README.md

```diff
@@ -0,0 +1,17 @@
+# Open Build Service at irill8.siege.inria.fr
```

Comment:
This salt instance is doing other things. Please add something on the top of this file!

Reply:
Done